tradeexecutor/visual/equity_curve.py

```diff
@@ -125,3 +126,13 @@ def calculate_aggregate_returns(equity_curve: pd.Series, freq: str = "BM"):
     return sampled.pct_change()
 
 
+def get_daily_returns(state: State) -> (pd.Series | None):
+    """Used for advanced statistics
+
+    :returns:
+        If valid state provided, returns are returned as business day frequency, else None"""
+    if isinstance(state, State):
+        equity_curve = calculate_equity_curve(state)
+        return calculate_aggregate_returns(equity_curve, freq="B")
```

Comment:
I think "business day" calculates only money-friday - so there is gap at Saturday, sunday

Reply:
Changed to `freq="D"` for calendar day frequency